### PR TITLE
Revive

### PR DESCRIPTION
**Pull request by PrimaMateria**
Hi @medwig 

Thank you for this tool. I am using it already for some time. Over the time, I made some dirty hacks to fit it to my workflow, and now I have finally found some time to clean it up.

The significant change is that you can define different type of mappings between Watson tags and target JIRA issues.
Besides that 
- the UI is updated
- instead of `.netrc` use app specific config file
- init command is updated

The application should still be backward compatible with your version.
Please let me know if you would like to merge it to your repo. 

I didn't add any unit tests, so maybe it would be also a good idea to wait before releasing it. I will be testing in upcoming days.

### CODE REVIEW

**Comment by PrimaMateria** [comment]
@medwig kind reminder

**Comment by medwig** [comment]
@PrimaMateria I'm very sorry for such a delayed response, my new job didn't require time logs so I forgot all about this project.

Thanks for putting up this pr, it looks great! I'll give it a review and let's get it merged :)

File: README.md [hunk 1]
```diff
@@ -1,58 +1,145 @@
...
+For any Watson log, which doesn't match any of the mappings, the JIRA issue number will be tried to be resolved from the project name.
+
+**Watson example:** `watson add -f 10:00 -t 11:00 JIRA-123 +investigation`
+
+### Full config example
+
+```
+jira:
+  server: http://localhost:8080
+  cookie: atlassian.xsrf.token=BEHZ-5GE9-RXNS-7J78_bfa98881ae96448d36fdaa94f2b3ac6b8f205885_lout; JSESSIONID=51D8547A4C356A8355F8FDAF7CC97D51
```
Comment (medwig) [inline README.md]: Likely shouldn't include your token eh?

Comment (medwig) [reply README.md]: I redacted it

**Comment by medwig** [comment]
Since it took me forever to look at this, I don't think it's fair to ask for any (minor) revisions. It's working in my manual tests so I'm going to merge and bump the version. I'll add tests before releasing the new version on pypi.

**Pull request merged**